test/spec/properties.js

```diff
@@ -73,6 +73,18 @@ describe('properties', function() {
       expect(inheritedAnyProperty).to.exist;
     });
 
+
+    it('should NOT add already defined property without redefine', function() {
+
+      // when
+      var getType = function() {
+        model.getType('props:BaseWithAlreadyDefinedId');
+      };
+
+      // then
+      expect(getType).to.throw(Error);
```

Comment:
TODO: Verify which error is thrown `expect(getType).to.throw(/some erorr message matcher/)`.

Why do we blow up late, i.e. on instantiation only? I'd assume we'd blow up when creating the package.

Reply:
> TODO: Verify which error is thrown expect(getType).to.throw(/some erorr message matcher/).

[Changed](https://github.com/bpmn-io/moddle/pull/39/commits/7bdb05fa5f70fc9df6e15d485378b63af85c3c61). Compares the whole exception string now.

> Why do we blow up late, i.e. on instantiation only? I'd assume we'd blow up when creating the package.

Yep, I'd also prefere earlier exception. If not `new Moddle()`, then latest in `Moddle.create()`.
This needs some kind of verification though. Maybe we should address this in a seperate issue?

This also applies to #38.